content/en/metrics/summary.md

```diff
@@ -143,10 +143,12 @@ You can configure tags using the bulk metric tag configuration button or the **M
 
 1. Click on your custom distribution metric name in the **Metrics Summary** table to open the metrics details side panel.
 2. Click the **Manage Tags** button to open the tag configuration modal.
-3. Click the **Custom...** tab to customize the tags you'd like to keep available for query. Tag configurations are _allowlists_ of the tags you'd like to keep. 
-4. Preview the effects of your proposed allowlist with the cardinality estimator before selecting **Save**.
+3. There are two ways in which you can configure your tags:
```

Comment:
Duplicating instructions and information across multiple pages can lead to maintenance issues in the future. How about redirecting customers to the more in-depth instructions you list in the Metrics without Limits page?
```suggestion

For more information on tag configuration, see the [Metrics without Limits][X] documentation.
```
Add the [X] link at the bottom of the page

Reply:
That makes sense! Thank you for pointing that out :)